tests/dump_input/vlan/appl_db.json

```diff
@@ -0,0 +1,56 @@
+{	
+    "VLAN_TABLE:Vlan2": {
+        "admin_status": "up",
+        "host_ifname": "",
+        "mac": "7c:fe:90:f5:36:40",
+        "mtu": "9100"
+        },
+    "VLAN_TABLE:Vlan3": {
```

Comment:
Align, the indentation in the JSON files. Either use spaces or tabs. FYI, If you are using VS Code, this should be a one click operation

Reply:
Assuming you mean that } should align with the section head, I fixed it for config, appl & state, the way asic already is.